src/multi_layer_chunker.py

```
@@ -481,6 +515,208 @@ def _create_layer3_basic(self, extracted_doc) -> List[Chunk]:
+            batch_input = [(contexts_text, title) for _, contexts_text, title in sections_data]
+
+            # Use min_text_length=0 because:
+            # - Chunk contexts (when available) are intentionally short but comprehensive
+            # - Raw chunk content (when contexts not available) still covers 100% of section
+            # Both approaches eliminate truncation problem
+            summaries = self.summary_generator.generate_batch_summaries(
+                batch_input, min_text_length=0
+            )
+
+            # Update sections with summaries
+            for (section_idx, _, _), summary in zip(sections_data, summaries):
```

Ruff (0.14.3)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)